graphql/input.go

```diff
@@ -0,0 +1,53 @@
+package graphql
+
+import (
+	"context"
+	"errors"
+	"reflect"
+)
+
+const unmarshalInputCtx key = "unmarshal_input_context"
+
+func BuildMap(unmarshaler ...interface{}) map[reflect.Type]reflect.Value {
```

Comment:
Please, add godoc.

Also, what do you think about returning an error instead of panic? 
```suggestion
func BuildUnmarshalerMap(unmarshaler ...interface{}) map[reflect.Type]reflect.Value {
```